### PR TITLE
disable kafka property in agent.config

### PR DESCRIPTION
**Pull request by xbkaishui**
disable kafka property for agent.config file 
#5300

### CODE REVIEW

**Comment by wu-sheng** [comment]
This breaks the e2e. You can't simply remove it.

**Comment by xbkaishui** [comment]
Why the tests depends on this file, is it better to depends a  file in the test/resources ?

**Comment by wu-sheng** [comment]
> Why the tests depends on this file, is it better to depends a file in the test/resources ?

I am describing the current status, not a requirement. The point is, you need to fix that part in the pull request too.

**Comment by xbkaishui** [comment]
Ok let me check. will give back later

**Comment by wu-sheng** [comment]
This is a part of Kafka e2e. You could find the process in `e2e.kafka.yaml` file.

**Comment by xbkaishui** [comment]
Ok Thanks

**Comment by xbkaishui** [comment]
Hi Wu Sheng, I have fixed the e2e test,  but have no idea why the codecov is fail, can you help me?  thanks

**Comment by wu-sheng** [comment]
> Hi Wu Sheng, I have fixed the e2e test, but have no idea why the codecov is fail, can you help me? thanks

Don't worry, it is fine.

File: test/e2e/e2e-test/docker/kafka/Dockerfile.provider [hunk 1]
```diff
@@ -25,6 +25,8 @@ ADD ${DIST_PACKAGE} .
 RUN tar -zxf apache-skywalking-apm*.gz --strip=1 || true
 RUN mv apache-skywalking-apm-bin/* . || true
 
+RUN sed -i 's/# plugin.kafka.bootstrap_servers/plugin.kafka.bootstrap_servers/g' agent/config/agent.config
```
Comment (wu-sheng) [inline test/e2e/e2e-test/docker/kafka/Dockerfile.provider:28]: Is this way good for you? `Check the agent.config` in the e2e process.

**Comment by dmsolr** [comment]
Thanks. And need to update document(how to enable Kafka reporter part)

File: test/e2e/e2e-test/docker/kafka/Dockerfile.provider (hunk 1 above)
Comment (dmsolr) [reply test/e2e/e2e-test/docker/kafka/Dockerfile.provider:28]: Suggestion:
```bash
sed -i 's/#\s*plugin.kafka.bootstrap_servers/plugin.kafka.bootstrap_servers/g' agent/config/agent.config
```

**Pull request merged**